### PR TITLE
Ci build workflow v3

### PR DESCRIPTION
**Pull request by lzadjsf**
Pull request has been re-created due to source branch has been changed.

### CODE REVIEW

**Comment by lzadjsf** [comment]
Try to re-run actions by close re-open PR.

**Comment by lzadjsf** [comment]
Re-opened to re-run actions check due to runner label has been changed to separate from Organisation runner.

**Comment by lzadjsf** [comment]
Closed. Have to be re-opened, due to check changes.

**Comment by lzadjsf** [comment]
Re-opened to re-run actions check due to check script has been removed at this moment, will be re-injected at future steps.

**Comment by lzadjsf** [comment]
@kozyrevdev could you please review this PR. All notes has been implemented.

File: .docker/Dockerfile-chain-dev [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN apt-get update && apt-get install -y git curl libssl-dev llvm pkg-config libclang-dev clang git make cmake
+
+ENV CARGO_HOME="/cargo-home"
+ENV PATH="/cargo-home/bin:$PATH"
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain none
+
+ARG RUST_TOOLCHAIN=nightly-2022-05-11
+ARG POLKA_VERSION=release-v0.9.24
```
Comment (kozyrevdev) [inline .docker/Dockerfile-chain-dev:14]: если это --dev режим, то зачем polkadot зависимость?

File: tests/scripts/readyness.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api');
```
Comment (kozyrevdev) [inline tests/scripts/readyness.js:1]: слишком обобщеное название скрипта

**Pull request merged**